### DISCUSSION TITLE
custom folder name

### DISCUSSION THREAD

**Discussion by 3154311518** [pufferpanel/pufferpanel#1173] [Ideas]
Custom created server folder name
This makes it easier to manage files
Otherwise, it will be hard to tell which is the subserver folder

**Comment by nepcore** [comment]
You shouldn't be directly messing around in the files in the first place, use the web file manager or the panels integrated SFTP server, otherwise the panel can't make sure that you don't break your servers for example by making file permissions prevent the panel and servers run through it from accessing certain files

As such there really isn't much need for human friendly directory names, which would also be troublesome in their own ways with things like different file systems not allowing different characters in folder and file names, name collisions, etc

**Reply by 3154311518** [reply]
I'm not very used to using SFTP
